client/src/components/MainPage/TranslationsPage/DonateModal/AudioTranslation/index.jsx

```diff
@@ -18,14 +41,27 @@ class AudioTranslation extends Component {
     this.setState({ selectedFile: files[0] });
   };
 
+  stop = () => {
+    const { recording } = this.state;
+    if (recording) {
+      const { player } = this.refs;
+      this.setState({ recording: false });
+      this.mediaRecorder.stop();
+      const audioBlob = new Blob(this.chunks);
+      const audio = new File(this.chunks, 'record.mp3', { type: 'audio', lastModified: Date.now() });
+      const audioUrl = URL.createObjectURL(audioBlob);
+      player.src = audioUrl;
+      player.classList.remove('no__record');
```

Comment:
@HemaSAli 
Could you add and remove the no__record class to the player element in render(), using this.state.recording, instead of manipulating the classList directly?  That might be a cleaner division of logic / presentation.

Reply:
@tomduggan85 , yeah yeah I got it, I will do it :+1: